apis/v1alpha1/instrumentation_webhook.go

```diff
@@ -64,26 +65,86 @@ func (r *Instrumentation) Default() {
 			r.Spec.Java.Image = val
 		}
 	}
+	if r.Spec.Java.Resources.Limits == nil {
+		r.Spec.Java.Resources.Limits = corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("500m"),
+			corev1.ResourceMemory: resource.MustParse("64Mi"),
```

Comment:
How were these values chosen (same goes for the rest of the languages?)

Reply:
They are based on the container image size.